debian/copy_installer.sh

```diff
@@ -1,9 +1,19 @@
 if [[ x"$1" =~ x"v2" ]]
 then
-   echo "Copy python3-saithrift.install as python-saithrift.install"
-   cp ./debian/installerFiles/python3-saithrift.install ./debian/python-saithrift.install
+   if [ $(lsb_release -sr) -eq 12 ]
+   then
+      echo "Copy python3-saithrift-bookworm.install as python-saithrift.install for Ver.12 releases"
+      cp ./debian/installerFiles/python3-saithrift-bookworm.install ./debian/python-saithrift.install
+   else
+      echo "Copy python3-saithrift.install as python-saithrift.install"
+      cp ./debian/installerFiles/python3-saithrift.install ./debian/python-saithrift.install
+   fi
 else
-   if [ $(lsb_release -sr) -ge 11 ]
+   if [ $(lsb_release -sr) -eq 12 ]
```

Comment:
I think we can make python3-saithrift-bookworm.install default for all the version >= 12. What do you think?

Reply:
Sure,  I will make python3-saithrift-bookworm.install default for version >= 12